### ISSUE TITLE
No meshes to process!

### ISSUE THREAD

**Issue by jhwang7628** [andyp123/blender_vhacd#6]
I am getting `No meshes to process!` error/warning in Blender 2.82a (see the warning message down below in yellow). Anyone has the same problem? The mesh is selected.

![image](https://user-images.githubusercontent.com/7445300/81843525-8f9b1500-9502-11ea-8c76-e6bd26442927.png)

**Comment by grossyoan** [comment]
I have the same issue. I did some testings, and the only way to get it working on my Mac was by using Bootcamp and Blender 2.8.

**Comment by marcofugaro** [comment]
Yeah I have the same problem, currently this plugin doesn't work on macos.

**Comment by matt-d-rat** [comment]
I have the same issue on macOS and ~~Windows~~. Blender v2.83 LTS.

Edit: I got it working on my Windows machine by using the No OpenCL executable.

**Comment by ethanjyx** [comment]
Same issue on Mac OS with Blender v2.91.2

**Comment by king40168** [comment]
same issue on windows with Blender v2.92.0
